import_map.json

```diff
@@ -10,7 +10,7 @@
     "https://deno.land/std@0.150.0/": "https://deno.land/std@0.181.0/",
     "https://deno.land/std@0.152.0/": "https://deno.land/std@0.181.0/",
     "https://deno.land/std@0.164.0/": "https://deno.land/std@0.181.0/",
-    "$fresh/": "https://deno.land/x/fresh@1.1.4/",
+    "$fresh/": "https://denopkg.com/deco-cx/fresh@1.3.0/",
```

Comment:
out ouf curiosity, why did you need that?

Reply:
It was outdated with two things

1. Now middlewares receives the `destination` property which tells if the next route will be either internal, static or a "normal" route, then I could use it to skip config resolution
2. the `config` property was not being used from fresh on manifest (which was causing some checks to fail)